Design-Process.md

```diff
@@ -0,0 +1,194 @@
+* Ends when the feature is [implemented](#implemented-feature) as part of a C# release
+* Ends if the feature is reconsidered at an LDM and then [rejected](#rejected-feature)
+
+### Implemented feature
+
+Once a feature has been implemented in the [Roslyn](https://github.com/dotnet/roslyn) C# compiler and been released as part of an official C# release, it is considered implemented.
+At this point, it will have a complete speclet available in the [proposals/csharp-\<release version\>](proposals) folder (note that some older C# features, particularly the C# 7.X
+and prior features, did not follow this, and have incomplete or non-existent speclets). At this point, the issue will be labeled `Implemented Needs ECMA Specification`, but it will
+not be closed until the ECMA-334 specification is updated with the feature. This can take some time; the ECMA-334 committee is working on catching up as fast as they can, but is
+several years behind the language implementation.
+
+#### Lifecycle
+
+* Starts when a [triaged feature](#triaged-feature) is shipped as part of a C# release
+* Ends when the feature is fully incorporated into a version of the ECMA-334 specification
+
+### ECMA-specified feature
+
+At this point, the feature has been fully incorporated by ECMA-TC49-TG2, the C# standards committee, into the
+[official C# ECMA specification](https://github.com/dotnet/csharpstandard/). When this happens, we close the issue as completed, and all development work on the feature is
+complete.
+
+#### Lifecycle
+
+* Starts when an [implemented feature](#implemented-feature) is shipped as part of a C# release
+* This is the final state for a feature that is included in C#, no further state changes occur
+
+### Rejected feature
+
+When a feature is explicitly considered during an LDM, and the LDT decides as a group to reject it, it moves to this state. At this point, close the champion issue as not planned
+and set the milestone to `Likely Never`. It's not impossible for an issue to be pulled back out of this state and included in the language in the future, but generally, this state
+means that the feature will never be part of C#.
+
+#### Lifecycle
+
+* Starts when a [championed feature](#championed-feature) is considered at LDM and rejected
+* While it is possible that some rejected features end up getting reconsidered, this is generally the final state for language features that are explicitly considered and
+  rejected during LDM
+
+## Language Design Team processes
+
+These are various processes and actions taken by LDT members during the development of a feature. Community members should not perform these actions unless invited to do so
+by an LDT member.
+
+### Steps to move a [Discussion](#proposed-feature) to a [Champion feature](#championed-feature)
+
+When an LDT member decides to champion a discussion, they take the following steps:
```

Comment:
Think we should outline what content we want in the issue. Too often I see effectively a copy of the original speclet in the issue. That is not always kept up to date as the speclet evolves and this can lead to confusion. 

My personal preference is that issues be a quick summary of the issue (broad strokes) but point to the proposal for the actual details of the feature.

Reply:
That's what the template updates are intended to convey, but I'll put that here too.